Gemfile

```diff
@@ -40,6 +40,9 @@ gem 'jbuilder', '~> 2.5'
 gem 'rubocop', '~> 0.55.0', require: false
 gem 'rubocop-rspec', '1.10.0'
 
+# Devise to authenticate users
+gem 'devise', git: 'git://github.com/plataformatec/devise.git'
```

Comment:
You don't need to specify the git repo, but you should specify the gem version!!